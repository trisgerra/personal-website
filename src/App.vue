<template>
  <div id="app">
    <nav-bar></nav-bar>
    <router-view/>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';
import NavBar from './components/NavBar/NavBar.vue';
@Component({
  components: {
    NavBar,
  },
})
export default class App extends Vue {

}

</script>


<style lang="scss">
</style>
