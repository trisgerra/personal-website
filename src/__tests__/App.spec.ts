import App from '../App.vue';
import createWrapper from '@/utils/CreateWrapper';
import { Wrapper } from '@vue/test-utils';
describe('App', () => {
    it('should render as expected', () => {
        const wrapper: Wrapper<App> = createWrapper(App);
        expect(wrapper.element).toMatchSnapshot();
    });
})