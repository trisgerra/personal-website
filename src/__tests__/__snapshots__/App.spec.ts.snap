// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render as expected 1`] = `
<div
  id="app"
>
  <nav
    aria-label="main navigation"
    class="navbar"
    role="navigation"
  >
    <div
      class="navbar-brand"
    >
      <a
        class="menu-item router-link-exact-active router-link-active"
        href="#/"
      >
        <div
          class="navbar-item navbar-logo"
        >
          <img
            alt="ADG Official"
            height="38"
            src="../../assets/logo.png"
          />
        </div>
      </a>
       
      <a
        aria-expanded="false"
        aria-label="menu"
        class="navbar-burger"
        role="button"
      >
        <span
          aria-hidden="true"
        />
         
        <span
          aria-hidden="true"
        />
         
        <span
          aria-hidden="true"
        />
      </a>
    </div>
     
    <div
      class="navbar-menu"
    >
      <div
        class="navbar-start"
      >
        <a
          class="menu-item router-link-exact-active router-link-active"
          href="#/"
        >
          <a
            class="navbar-item navbar-router"
          >
            Home
          </a>
        </a>
        <a
          class="menu-item"
          href="#/erace"
        >
          <a
            class="navbar-item navbar-router"
          >
            ERace
          </a>
        </a>
      </div>
    </div>
  </nav>
   
  <div
    class="home"
  >
    <div>
      <div
        class="container desktop is-hidden-touch"
      >
        <div
          class="absolute columns"
        >
          <div>
            <img
              alt="profile image"
              class="profile"
              src="../../../assets/profile2.jpg"
            />
          </div>
           
          <div
            class="column text-container"
          >
            <h2
              class="title"
            >
              Andrea DG &lt;website/&gt;
            </h2>
             
            <h3
              class="subtitle"
            >
              WHO AM I?
            </h3>
             
            <p
              class="description"
            >
              I'm 
              <b>
                simply
              </b>
               a Software developer with a passion for 
              <b>
                new technologies
              </b>
               and the 
              <b>
                design
              </b>
               .
            </p>
          </div>
        </div>
         
        <nav
          class="columns"
        >
          <div
            class="column first"
          />
           
          <div
            class="column"
          />
           
          <div
            class="column"
          />
           
          <div
            class="column"
          />
           
          <div
            class="column"
          />
           
          <div
            class="column"
          />
        </nav>
      </div>
       
      <div
        class="container mobile is-hidden-desktop"
      >
        <div
          class="columns"
        >
          <div
            class="column is-12-mobile"
          >
            <img
              alt="profile image"
              class="profile-mobile"
              src="../../../assets/profile2.jpg"
            />
          </div>
           
          <div
            class="column text-container"
          >
            <h2
              class="title"
            >
              Andrea DG &lt;website/&gt;
            </h2>
             
            <h3
              class="subtitle"
            >
              WHO AM I?
            </h3>
             
            <p
              class="description"
            >
              I'm 
              <b>
                simply
              </b>
               a Software developer with a passion for 
              <b>
                new technologies
              </b>
               and the 
              <b>
                design
              </b>
               .
            </p>
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="card-container"
    >
      <div
        class="columns"
      >
        <div
          class="column home-card"
        >
          <div
            class="card column dark"
          >
            <div
              class="card-content"
            >
              <h3
                class="card-title"
              >
                WORK EXPERIENCE
              </h3>
               
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  2012
                </b>
                 - iOS Internship
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  2013
                </b>
                 - iOS Developer
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  2014
                </b>
                 - Mobile Developer
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  2015
                </b>
                 - Mobile Developer & Analyst
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  2016
                </b>
                 - Mobile Developer, Analyst Designer
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  2017
                </b>
                 - Mobile Lead
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  2018
                </b>
                 - Frontend Developer
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  Now
                </b>
                 - Senior Frontend Developer
              </p>
            </div>
          </div>
        </div>
        <div
          class="column home-card"
        >
          <div
            class="card column"
          >
            <div
              class="card-content"
            >
              <h3
                class="card-title"
              >
                SKILLS
              </h3>
               
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  iOS
                </b>
                 - [Obj-C, Swift, React Native]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  Android
                </b>
                 - [Java, Swift, React Native]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  Web
                </b>
                 - [Vue, React]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  Unit Testing
                </b>
                 - [Jest]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  E2E Testing
                </b>
                 - [Cypress]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #151616;"
              >
                <b>
                  Design
                </b>
                 - [Invision Studio, Sketch, Photoshop]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #151616;"
              >
                <b>
                  CI/CD
                </b>
                 - [Travis CI, Circle CI, CodeClimate, Jenkins, Heroku]
              </p>
              <p
                class="item"
                style="border-left: 3px solid #151616;"
              >
                <b>
                  Devops
                </b>
                 - [Docker, Kubernetes]
              </p>
            </div>
          </div>
        </div>
        <div
          class="column home-card"
        >
          <div
            class="card column dark"
          >
            <div
              class="card-content"
            >
              <h3
                class="card-title"
              >
                HOBBIES
              </h3>
               
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  Vaping
                </b>
              </p>
              <p
                class="item"
                style="border-left: 3px solid #FF0000;"
              >
                <b>
                  Drive simulator
                </b>
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  Fitness
                </b>
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  Politics
                </b>
              </p>
              <p
                class="item"
                style="border-left: 3px solid #00D1FF;"
              >
                <b>
                  Cryptocurrencies
                </b>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="square-container"
    >
      <div
        class="square-row"
      >
        <div
          class="columns"
        >
          <div
            class="column square-card"
          >
            <div
              class="square-card"
              style="background-color: rgb(21, 22, 22);"
            >
              <a
                href="https://github.com/trisgerra"
                target="_blank"
              >
                <h3>
                  GITHUB
                </h3>
              </a>
            </div>
          </div>
          <div
            class="column square-card"
          >
            <div
              class="square-card"
              style="background-color: rgb(45, 47, 47);"
            >
              <a
                href="https://www.linkedin.com/in/a-digiacomo/"
                target="_blank"
              >
                <h3>
                  LINKEDIN
                </h3>
              </a>
            </div>
          </div>
          <div
            class="column square-card"
          >
            <div
              class="square-card"
              style="background-color: rgb(73, 74, 74);"
            >
              <a
                href="https://www.twitch.tv/asraces"
                target="_blank"
              >
                <h3>
                  TWITCH
                </h3>
              </a>
            </div>
          </div>
          <div
            class="column square-card"
          >
            <div
              class="square-card"
              style="background-color: rgb(45, 47, 47);"
            >
              <a
                href="mailto:andrea.digiacomo@live.it"
                target="_blank"
              >
                <h3>
                  CONTACT
                </h3>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
